extension/content/index.html

```diff
@@ -24,23 +24,51 @@
     </div>
 
     <div id="server-settings">
-      <div>
-        <span class="radio">
-          <input type="radio"
-                 id="use-main"
-                 name="server-radio"
-                 value="main"
-                 checked/>
-          <label for="use-main">Use active configuration</label>
-        </span>
-        <span class="radio">
-          <input type="radio"
-                 id="use-preview"
-                 name="server-radio"
-                 value="main-preview"/>
-          <label for="use-preview">Use preview configuration</label>
-        </span>
-      </div>
+      <table>
```

Comment:
So I dont think this is worth blocking review, but aside from the main table (which we dont use a table) I think we should probably try to avoid using tables for layout.

Reply:
Sure, I'll look at that in future.